CountToMillion/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include <string>
+#include <iostream>
+#include "Counter.h"
+#include "ManagedEvent.h"
+#include "SharedObject.h"
+
+const std::wstring firstEventName = L"FirstCounterEvent";
+const std::wstring secondEventName = L"SecondCounterEvent";
+const std::wstring sharedMutexName = L"SharedMutex";
```

Comment:
These should be declared in `main` as `static const`s, since they don't apply to anywhere but there.

- [ ] Move into main